src/Routers/routes/ConsultationRoutes.tsx

```diff
@@ -15,6 +16,14 @@ const consultationRoutes: AppRoutes = {
         patientId={patientId}
       />
     ),
+  "/facility/:facilityId/patient/:patientId/encounter/:encounterId/medicines/administrations/print":
```

Comment:
org/orgId route? 🤔

Reply:
Currently, facilityId is required for fetching medication administrations; so let's push this as first iteration and add support for print outs in org routes in a separate PR.